### PR TITLE
Nav 41 api stubs + auth updates

### PR DESCRIPTION
**Pull request by tomeksabala**
Implements stubs from: https://documenter.getpostman.com/view/13770205/UVBzm9T4

### CODE REVIEW

**Review by Manoj-nathwani — APPROVED** [review]

File: navigator_api/main.py [hunk 1]
```diff
@@ -38,98 +26,111 @@ def user_details():
...
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa",
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa"
+        ],
+        "currentTask": {
+            "id": "asdfasdfa",
+            "skipped": False,
```
Comment (Manoj-nathwani) [inline navigator_api/main.py:104]: @tomeksabala why is `skipped` here?
This is the current task, so it cannot already have a skipped state.
I think it was left here by accident

Comment (tomeksabala) [reply navigator_api/main.py:104]: It's task details json. Even though current task can never be "skipped" it's included here for consistency with every other task details instance.

Comment (Manoj-nathwani) [reply navigator_api/main.py:104]: ok

File: navigator_api/main.py [hunk 2]
```diff
@@ -38,98 +26,111 @@ def user_details():
...
+        "tasks": [
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa",
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa",
+            "aaabbbbcc",
+            "dddeeefff",
+            "asdfasdfa"
```
Comment (Manoj-nathwani) [inline navigator_api/main.py:100]: Is this a list of task ids?
Why do we have this?

Comment (Manoj-nathwani) [inline navigator_api/main.py:100]: @tomeksabala Is this a list of task ids? Why do we have this?

Comment (tomeksabala) [reply navigator_api/main.py:100]: That's the link of previous tasks user has visited.

**Pull request merged**